patches/perfetto/build_win_fix_compilation_error_with_msstl.patch

```diff
@@ -0,0 +1,58 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Deepak Mohan <hop2deep@gmail.com>
+Date: Thu, 24 Oct 2019 17:58:10 -0700
+Subject: build,[win]: fix compilation error with MSSTL
+
+Moves the accessor definition from header file to avoid the following
```

Comment:
I will upstream this, but will be addressing https://github.com/electron/electron/issues/20195 shortly which should remove this patch as well.

Reply:
Upstream CL https://android-review.googlesource.com/c/platform/external/perfetto/+/1152120